rero_ils/modules/items/api.py

```diff
@@ -238,6 +267,37 @@ def get_pendings_loans(cls, library_pid):
         for loan in results:
             yield Loan.get_record_by_pid(loan.loan_pid)
 
+    @classmethod
+    def get_loaned_loans(cls, patron_pid):
+        """."""
+        # check library exists
+        patron = Patron.get_record_by_pid(patron_pid)
+        if not patron:
+            raise Exception('Invalid Patron PID')
+
+        results = current_circulation.loan_search\
+            .source(['loan_pid'])\
+            .params(preserve_order=True)\
+            .filter('term', state='ITEM_ON_LOAN')\
+            .filter('term', patron_pid=patron_pid)\
+            .sort({'transaction_date': {'order': 'asc'}})\
+            .scan()
+        for loan in results:
+            yield Loan.get_record_by_pid(loan.loan_pid)
+
+    @classmethod
+    def get_loaned_items(cls, patron_pid):
```

Comment:
Better function name?  `loaned = checked-out ` ... get_checked_out_items for example.